### ISSUE TITLE
Ordering of releases from unannotated tags

### ISSUE THREAD

**Issue by schtiefel** [gogs/gogs#5549]
Releases that are automatically generated when pushing *unannotated tags*, are listed in a seemingly random order.
What kind of ordering is this?

For a release overview one (at least I) would expect a date-order. Wouldn't this be more intuitive?
Date-ordering of unannotated tags can e.g. be achieved via `git tag --sort=-committerdate`, which takes the date of the corresponding commit per tag.

**Comment by schtiefel** [comment]
I'm sorry, I thought I did. What did I miss?

**Comment by Unknwon** [comment]
Read my last comment.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by schtiefel** [comment]
Yeah, what part of the [Feature request](https://github.com/gogs/gogs/issues/new?template=feature_request.md) template didn't I respect?

---

The issue will be closed without any explanation if it does not satisfy any of following requirements:

1. Please speak English, we have forum in [中文](https://discuss.gogs.io/c/getting-help/getting-help-chinese).
_→ check_
2. Please post questions or config/deploy problems on our forum: https://discuss.gogs.io, here are bugs and feature requests only.
_→ does not apply_
3. Please do not end your title with a question mark or period.
_→ check_
4. Please take a moment to search that an issue doesn't already exist.
→ there was one other issue regarding release ordering _with pagination_ #4482)
_→ check_

**You MUST delete above content including this line before posting; too lazy to take this action considered invalid report.**
_→ check_

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
> Releases that are automatically generated when pushing unannotated tags, are listed in a seemingly random order.

_→ check_

**Describe the solution you'd like**
A clear and concise description of what you want to happen.
> I [..] would expect a date-order. [..] can [..] be achieved via `git tag --sort=-committerdate`

_→ check_

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.
_→ does not apply (imho)_

**Additional context**
Add any other context or screenshots about the feature request here.
→ Everyone here knows how the release page looks like. It's no rarely occuring, hard-to-reproduce bug.
_→ does not apply (imho)_

**Comment by Unknwon** [comment]
1. What I saw is a random paragraph, not following any templates.
2. Does not look like a feature request, should be a bug report.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by schtiefel** [comment]
Even if it was just a random paragraph in the beginning, it is not since my last comment.
Even if you thought this should be a bug report, and thus some template information is missing (Gogs Version: 0.11.66.0916), you could have kindly asked for this information instead of just closing the issue.

Instead, you behaved like you don't care about feedback - so, ignore it or not - I'll no longer care either.

**Comment by Unknwon** [comment]
I've been kind enough up until this point.
